app/chitter.rb

```diff
@@ -0,0 +1,62 @@
+ENV['RACK_ENV'] ||= 'development'
+require 'sinatra/base'
+require 'sinatra/flash'
+require './app/data_mapper_setup'
+
+class Chitter < Sinatra::Base
+  enable :sessions
+  set :session_secret, 'super secret'
+  register Sinatra::Flash
+  use Rack::MethodOverride
+
+  helpers do
+    def current_user
+      @current_user ||= User.get(session[:user_id])
+    end
+  end
+
+  get '/sessions/new' do
+    erb(:'sessions/new')
+  end
+
+  post '/sessions/new' do
+    email = params[:email]
+    password = params[:password]
+    if user = User.authenticate(email: email, password: password)
```

Comment:
Lint/AssignmentInCondition: Assignment in condition - you probably meant to use `==`. (https://github.com/bbatsov/ruby-style-guide#safe-assignment-in-condition)